nodes/arduino_handler.py

```diff
@@ -383,6 +383,10 @@ def connect_serial(serial_connection=None):
             trace('arduino_handler serial write %d bytes: >%s<', len(message), message.replace('\n',''))
             # Read
             buf = serial_connection.readline()
+            # handle reading nothing, happens when serial port times out
```

Comment:
If we let process_message run, it will do the sleep at the end of the while loop